docs/source/en/model_doc/detr.md

```diff
@@ -120,26 +171,27 @@ required for the training to be possible, since the Hungarian matching is comput
 - The size of the images will determine the amount of memory being used, and will thus determine the `batch_size`.
   It is advised to use a batch size of 2 per GPU. See [this Github thread](https://github.com/facebookresearch/detr/issues/150) for more info.
 
-There are three ways to instantiate a DETR model (depending on what you prefer):
+There are three other ways to instantiate a DETR model (depending on what you prefer):
```

Comment:
Put all 3 options in bullet points (everything in `## Notes` should be bulleted)